core/src/main/java/com/linecorp/armeria/server/http/auth/HttpAuthServiceBuilder.java

```diff
@@ -30,73 +32,53 @@
 import com.linecorp.armeria.server.Service;
 
 /**
- * Builds a new {@link HttpAuthService}.
+ * Builds a new {@link HttpAuthService}. {@link Authorizer}s are executed without short-circuiting -
+ * all authorizers will be executed regardless of the result of any individual one.
```

Comment:
Any authorizer returning ```true``` will allow the request to process. This is saying that all ```Authorizer#authorize``` methods will be called regardless. Previously, if an authorizer returned true, subsequent authorizers would not have authorize called at all.